### ISSUE TITLE
several tables have 0 rows when ETL complete

### ISSUE THREAD

**Issue by turbomam** [OHDSI/ETL-Synthea#28]
See also https://forums.ohdsi.org/t/does-etl-synthea-work/7335

**Comment by turbomam** [comment]
Just pulled today.  Running Postgres 11 and the ETL bulk load script on Ubuntu 18.

```
Performing ETL
Loading person
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 1189
Loading observation_period
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 1189
Loading visit_occurrence
INSERT 0 36985
Loading observation
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 0
Loading condition_occurrence
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 8750
Loading procedure_occurrence
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 0
Loading measurement
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 0
Loading drug exposure
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 0
Loading condition_era
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 7132
Loading drug_era
DROP SEQUENCE
CREATE SEQUENCE
INSERT 0 0
```

**Comment by turbomam** [comment]
I just used an older Athena vocabulary download (2019-05-21) and the 0-length tables aren't problem any more.  So this issue is specific to vocabularies I downloaded today.  I'll examine them more closely to see what's going on.

**Comment by trankhavy** [comment]
Hi,

I ran into the same issue and found your answer really useful. Can you please provide the download link for the older Athena vocabulary set?

**Comment by turbomam** [comment]
Hi, @trankhavy 

I don't think there's any permanent links for vocabulary downloads that would work outside of the creator's login session. I'm trying to think of a way to help you, but I'm sure you know that these files are pretty large, and I don't think we're supposed to redistribute them.  

Have you tried posting to the OHDSI forum?  It's quite active, including a lot of participation from high profile contributors like Christian Reich.  https://forums.ohdsi.org/

I'll reopen this case so others will see it, and I'll keep thinking if there's some other way I can help.  For what it's worth, I built my "older" vocab file (which is compatible with the Synthea ETL)  on 2019-05-21.  My newer unsuccessful one was built on 2019-07-06.  I just tried to restore the 2019-05-21 build from my Athena archive, but I was told that one of the components now requires a license.

-Mark
